src/coreclr/scripts/superpmi_setup.py

```diff
@@ -202,7 +249,8 @@ def sorter_by_size(pair):
         # Credit: https://stackoverflow.com/a/19859907
         dirs[:] = [d for d in dirs if d not in exclude_directories]
         for name in files:
-            if name in exclude_files:
+            # Make the exclude check case-insensitive
```

Comment:
I would convert all the file names to lower case outside the loop:

```py
exclude_files_lower = [filename.lower() for filename in exclude_files]
for name in files:
     if name.lower() in exclude_files_lower:
          continue
```

Reply:
I meant to have the `exclude_files_lower = [filename.lower() for filename in exclude_files]` outside of `for name in files:`

Reply:
otherwise we are doing `.lower()` on `exclude_files` for every file we iterate.

Reply:
well, it's not perf critical...